src/coder.cpp

```diff
@@ -253,13 +230,17 @@ int main(int argc, char * argv[]) {
     std::ifstream file_in;
     std::ofstream file_out;
     std::string fname;
+    size_t blocksize = DATA_BLOCK_LEN;
     int ret;
 
-    if (argc != 2 || ! strcmp(argv[1], "-h"))
-        return print_help(argv[0]);
+    if (argc < 2 || ! strcmp(argv[1], "-h"))
```

Comment:
It could be appropriate to print also for `argc >= 3`.

Reply:
Maybe?:
```c
if (argc < 2 || argc > 3 || ! strcmp(argv[1], "-h"))
```

Reply:
Oh, right :+1: